x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/flyout.tsx

```diff
@@ -9,15 +9,18 @@ import React, { useMemo, useState } from 'react';
 import { EuiFlyout, EuiFlyoutHeader, EuiTitle, EuiFlyoutBody } from '@elastic/eui';
 import { EuiSpacer, EuiTabs, EuiTab } from '@elastic/eui';
 import { MetadataTab } from './metadata/metadata';
+import type { InventoryItemType } from '../../../../../../common/inventory_models/types';
 import type { HostNodeRow } from '../../hooks/use_hosts_table';
 import { useUnifiedSearchContext } from '../../hooks/use_unified_search';
+import { ProcessesTab } from '../../../inventory_view/components/node_details/tabs/processes';
 
 interface Props {
   node: HostNodeRow;
   closeFlyout: () => void;
 }
 
-const flyoutTabs = [MetadataTab];
+const flyoutTabs = [MetadataTab, ProcessesTab('hostsView-flyout-tabs-processes')];
```

Comment:
As said in another suggestion, here we could create a new `ProcessesTab` with its own `data-subj-test` or use a spread operator. Also, maybe `MetadataTab` could be camelCased? It's actually an object.